### ISSUE TITLE
awscdk/construct-constructor-property does not recognize that a class extends Construct if the class is imported from a library

### ISSUE THREAD

**Issue by JohnnyOu** [ren-yamanashi/eslint-cdk-plugin#428]
### Feature Description

Rule `awscdk/construct-constructor-property` is only working as expected if the class that extends Construct is defined within the source code. If this class is imported from a library and `MyConstruct` extends the library class, then the rule does not properly flag the incorrect constructor param naming.

This could also be considered a bug, but I have decided to mark it as feature request because it has no false positive cases, and I'm not sure how much effort is required to implement this.

### Motivation

Without this feature, some classes have hidden ESLint errors that are not caught.

### Invalid Code Examples

```typescript
// LibraryConstruct extends Construct or extends another class which ultimately extends Construct
import { LibraryConstruct } from "some/library";

// Currently the rule does not recognize that MyConstruct actually extends Construct, so it does not flag error
export class MyConstruct extends LibraryConstruct {
  constructor(myScope: Construct, id: string) {
    super(myScope, id);
  }
}
```

### Valid Code Examples

All other current valid cases can remain. No new valid cases introduced.

### Implementation Details

- [x] I'm willing to submit a PR for this feature

**Comment by ren-yamanashi** [comment]
Thanks feature request! Wellcome to contribution!
